packages/initializer-jwt/src/index.ts

```diff
@@ -37,6 +39,8 @@ export default (
           expiresIn: config.controllers!.jwtExpiry || '14d',
           algorithm: config.controllers!.jwtAlgorithm || 'HS512',
         },
+        authCookieName:
```

Comment:
I think if authCookieName is not supplied, then the feature shouldn't be active. This `||` logic and default means that cookie is always going to be active, no?